### PR TITLE
Basic User profile

### PR DESCRIPTION
**Pull request by jefflau**

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

File: src/components/User/Avatar.js [hunk 1]
```diff
@@ -19,6 +20,14 @@ const Avatar = ({ href, src, className }) => {
         </a>
       </AvatarContainer>
     )
+  } else if (username) {
+    container = (
+      <AvatarContainer className={className}>
+        <Link to={`/user/${username}`}>
+          <img src={src} alt="avatar" />
```
Comment (makoto) [inline src/components/User/Avatar.js]: Possibly a bit too big. Maybe width 150px or something like that?

Screenshot 2019-07-05 at 14 47 03

File: src/App.js [hunk 2]
```diff
@@ -67,6 +68,7 @@ class App extends Component {
                 path="/event/:address/admin"
                 component={SingleEventAdmin}
               />
+              <Route path="/user/:username" component={UserProfile} />
```
Comment (makoto) [inline src/App.js]: Can we have link from somewhere? One idea is to link to /user/$username when clicking username from the header. If we do so, can you also add link to "Edit Profile" from the user profile page?

Comment (makoto) [reply src/App.js]: Also can we replace the link from the list of participants which currently link to twitter address?

File: src/components/UserProfile/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  const twitter = getSocialId(p.social, 'twitter')
+  return (
+    <UserProfileContainer>
+      <Avatar src={`https://avatars.io/twitter/${twitter}`} />
+      <H2>{p.username}</H2>
+      {twitter && (
+        <a href={`https://twitter.com/${twitter}`}>Twitter: {twitter}</a>
+      )}
+
+      <H3>Events Attended ({p.eventsAttended.length})</H3>
```
Comment (makoto) [inline src/components/UserProfile/index.js]: Is this list of events attended or the list of events I did RSVP?

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

File: src/components/UserProfile/index.js (hunk 3 above)
Comment (jefflau) [reply src/components/UserProfile/index.js]: RSVP'd

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr146-app-wearekickback.surge.sh

**Pull request merged**